packages/themes/default/styles/variables/_navbar.scss

```diff
@@ -0,0 +1,115 @@
+/* Navbar */
+
+//  Spacing
+$theme-site-navbar-padding-x:           16px !default;
+$theme-site-navbar-padding-y:           0 !default;
+$theme-site-navbar-brand-margin-y: 0 !default;
+$theme-site-navbar-brand-margin-x: 12px !default;
+
+// Colors
+
+// Dark
+$theme-site-navbar-dark-color:          rgba($white, .75) !default;
+$theme-site-navbar-dark-hover-color:    $white !default;
+$theme-site-navbar-dark-active-color:   $theme-site-navbar-dark-hover-color !default;
+
+// Light
+$theme-site-navbar-light-color:         rgba($black, .75) !default;
+$theme-site-navbar-light-hover-color:   $black !default;
+$theme-site-navbar-light-active-color:  $theme-site-navbar-light-hover-color !default;
+
+// Themes
+$theme-site-navbar-map: (
+  dark: (
+    default: $theme-site-navbar-dark-color,
+    hover: $theme-site-navbar-dark-hover-color,
+    active: $theme-site-navbar-dark-active-color
+  ),
+  light: (
```

Comment:
How would I override these colors now, to something other than the pre-defined dark/light values? The reason there were variables for each navbar  type is so they could be individually overridden (if need be) regardless of what type was selected

Reply:
From how I am reading this, it would be the same way you would have done it before, as they are still pulling from variable settings. I may be misunderstanding your question however. If I am misunderstanding, could you provide an example of the kind of change of which you are speaking?

With this change, there are still different navbar types, they have just been consolidated to dark and light, as the primary, secondary, and tertiary variables were all using the same default dark settings.